Source/SelfService/Web/documentation/cicd/setupAzurePipelines.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import React from 'react';
+import { useSnackbar } from 'notistack';
+
+import { HttpResponseApplications2 } from '../../api/api';
+import { DownloadButton } from '../../theme/downloadButton';
+import { getKubernetesServiceAccount, getContainerRegistry } from '../../api/cicd';
+type Props = {
+    application: HttpResponseApplications2
+    info: any
+};
+
+export const Doc: React.FunctionComponent<Props> = (props) => {
+    const { enqueueSnackbar } = useSnackbar();
+    const applicationID = props!.application.id;
+    const buttonServiceAccount = <DownloadButton
+        onClick={async (event: React.MouseEvent<HTMLElement>) => {
+            try {
+                const data = await getKubernetesServiceAccount(applicationID);
+                await navigator.clipboard.writeText(data);
+                enqueueSnackbar('Kubernetes service account copied to clipboard.');
+            } catch {
+                enqueueSnackbar('Failed to get data.', { variant: 'error' });
+            }
+        }}
+    >
+        Copy Credentials to clipboard
+    </DownloadButton>;
```

Comment:
The `<DownloadButton>`'s look a bit weird as you're just copying credentials, but I also know that mui v4 icons don't have the nice copy icon yet. In the future we should make a `<CopyButton>` for this. Just pointing this out as a paper trail https://app.asana.com/0/1200181647276434/1201312046888451

Reply:
Changed to ButtonText.